client/trino-client/src/main/java/okhttp3/internal/tls/LegacyHostnameVerifier.java

```diff
@@ -66,4 +75,129 @@ public boolean verify(String host, SSLSession session)
             return false;
         }
     }
+
+    private boolean verifyAsIpAddress(String host)
+    {
+        return VERIFY_AS_IP_ADDRESS.matcher(host).matches();
+    }
+
+    private boolean verifyHostname(String hostName, String pattern)
```

Comment:
If this is also a copy from OkHttp, can you add a comment, similar to the one in `DistinguishedNameParser.java`?